Platon/app/src/main/res/drawable/ic_add_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="M19,13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
+      android:fillColor="#000000"/>
```

Comment:
our design pattern generally requires "primary_light" for button image. Please double check